docs/guide/tutorials/custom-logs.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebarDepth: 0
+---
+# Deployment: Logging
+
+> How can you customize your Prefect logs?
+
+## Logging
+
+Logging is a crucial aspect of any production environment.  Prefect exposes a series of configuration settings and utility functions for customizing the behavior of your logs.
+
+### Adding your own logs
+
+The most common place you might want to add additional logs is within a custom Task.  There are two places you can access your Task's logger, depending on how you created the Task:
+
+- if your Task is implemented as a subclass of the `Task` class, the `self.logger` attribute contains your Task's logger
+- if your Task is implemented via the `task` decorator, you can access your logger from context: `logger = prefect.context.get("logger")`
+
+### Configuration
+
+Your [Prefect user configuration file](../core_concepts/configuration.html) provides one way to easily change how logs are presented to you when running locally.  In your user config file, simply add a section for logging with the following structure:
+```
+[logging]
+# The logging level: NOTSET, DEBUG, INFO, WARNING, ERROR, or CRITICAL
+level = "INFO"
+
+# The log format
+format = "[%(asctime)s] %(levelname)s - %(name)s | %(message)s"
+```
+
+Alternatively, you can set the following environment variables:
+```bash
+export PREFECT__LOGGING__LEVEL="INFO"
+export PREFECT__LOGGING__FORMAT="[%(asctime)s] %(levelname)s - %(name)s | %(message)s"
+```
+
+### Adding Handlers
```

Comment:
Can you add a blurb explaining what a handler is and why it's needed? Can be a short sentence or two.